src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -1,85 +1,87 @@
+import {getTopic} from '../../../../store/reducers/topic';
+import {
+    getOlapStats,
+    resetLoadingState as resetOlapLoadingState,
+} from '../../../../store/reducers/olapStats';
+import {
+    getOverview,
+    getOverviewBatched,
+    setCurrentOverviewPath,
+    setDataWasNotLoaded,
+} from '../../../../store/reducers/overview/overview';
+
 import {
     isEntityWithMergedImplementation,
     isColumnEntityType,
     isTableType,
     isPathTypeWithTopic,
 } from '../../utils/schema';
-//@ts-ignore
-import {
-    getSchema,
-    getSchemaBatched,
-    resetLoadingState,
-    selectSchemaMergedChildrenPaths,
-} from '../../../../store/reducers/schema/schema';
-import {getTopic} from '../../../../store/reducers/topic';
-//@ts-ignore
-import {
-    getOlapStats,
-    resetLoadingState as resetOlapLoadingState,
-} from '../../../../store/reducers/olapStats';
-import {useAutofetcher, useTypedSelector} from '../../../../utils/hooks';
+
+import {TopicInfo} from './TopicInfo';
+import {ChangefeedInfo} from './ChangefeedInfo';
+import {TableInfo} from './TableInfo';
 
 interface OverviewProps {
     type?: EPathType;
-    className?: string;
     tenantName?: string;
 }
 
-function Overview({type, tenantName, className}: OverviewProps) {
+function Overview({type, tenantName}: OverviewProps) {
     const dispatch = useDispatch();
 
+    const {autorefresh, currentSchemaPath} = useTypedSelector((state) => state.schema);
     const {
-        currentSchema: currentItem = {},
-        loading: schemaLoading,
-        wasLoaded,
-        autorefresh,
-        currentSchemaPath,
-        error,
-    } = useSelector((state: any) => state.schema);
-
-    const {data: {result: olapStats} = {result: undefined}, loading: olapStatsLoading} =
-        useTypedSelector((state) => state.olapStats);
+        data,
+        additionalData,
+        loading: overviewLoading,
+        wasLoaded: overviewWasLoaded,
+        error: overviewError,
+    } = useTypedSelector((state) => state.overview);
+    const {
+        data: {result: olapStats} = {result: undefined},
+        loading: olapStatsLoading,
+        wasLoaded: olapStatsWasLoaded,
+    } = useTypedSelector((state) => state.olapStats);
 
-    const loading = schemaLoading || olapStatsLoading;
+    const loading =
+        (overviewLoading && !overviewWasLoaded) || (olapStatsLoading && !olapStatsWasLoaded);
 
     const isEntityWithMergedImpl = isEntityWithMergedImplementation(type);
 
-    // There is a circular dependency here. Fetch data depends on children paths
-    // When data in store updated on fetch request,
-    // new object is set there, so source children array is updated
-    // This updates selector, the selector returns a new array, and data is fetched again
-    // To prevent it, shallowEqual, which compares array content, was added
+    // shalloEqual prevents rerenders when new schema data is loaded
```

Comment:
Here still could be data update when preloaded or previously loaded data (`schema` keeps data) is replaced with new data